lms/djangoapps/mobile_api/video_outlines/tests.py

```diff
@@ -362,6 +362,29 @@ class TestVideoSummaryList(TestVideoAPITestCase, MobileAuthTestMixin, MobileEnro
     """
     REVERSE_INFO = {'name': 'video-summary-list', 'params': ['course_id']}
 
+    def test_only_on_web(self):
+        self.login_and_enroll()
+
+        course_outline = self.api_response().data
+        self.assertEqual(len(course_outline), 0)
+        ItemFactory.create(
+            parent=self.unit,
+            category="video",
+            display_name=u"test video",
+        )
+
+        ItemFactory.create(
+            parent=self.unit,
+            category="video",
+            display_name=u"test video 2",
+            unavailable_on_mobile=True
+        )
+        course_outline = self.api_response().data
+        self.assertEqual(len(course_outline), 2)
+        self.assertEqual(course_outline[0]["summary"]["only_on_web"], False)
+        self.assertEqual(course_outline[1]["summary"]["only_on_web"], True)
```

Comment:
Hmm, it's probably fine to assume. I think the better thing to do then is to add a check that you have the correct video with the field we are testing:

`self.assertEqual(course_outline[0]["summary"]["only_on_web"], False)`
`self.assertEqual(course_outline[0]["summary"]["display_name"], u"test video")`
`self.assertEqual(course_outline[1]["summary"]["only_on_web"], True)`
`self.assertEqual(course_outline[0]["summary"]["display_name"], u"test video 2")`